### PR TITLE
Add Transcription Project Guide

### PR DESCRIPTION
**Pull request by snblickhan**
Added new Transcription Project Guide (content from [Google Docs](https://docs.google.com/document/d/1FYfg_2l-30vm6b0NekJxb0QGvYVmfESQBprSA1JGxiA/edit#heading=h.qus42ay60r7s))

Added custom navigation via edits to mkdocs.yml

### CODE REVIEW

File: docs/transcription-project-guide/beta-testing/beta-step-by-step.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# 5.2 Beta: step by step
+
+If you want your project to launch publicly, you must go through the review process. A public launch means that 1) we will send out a launch newsletter to our full volunteer community; and 2) your project will be marked as an ‘official’ Zooniverse project and listed at zooniverse.org/projects. If you do not want to launch publicly, you do not have to apply for beta review. Just build a project, set it to public, and send out your URL to your audiences. Please note that choosing to run a private project (and therefore skipping beta review) may limit our ability to assist you in later project stages – beta testing is a great way to catch issues with your project early on, while they are still easy to change.
+
+
+For an overview of the project review process, click [here](https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68/edit?usp=sharing).
```
Comment (eatyourgreens) [inline docs/transcription-project-guide/beta-testing/beta-step-by-step.md:6]: Small accessibility tip: avoid generic links BL text like 'here' or 'click here'. 

https://www.w3.org/QA/Tips/noClickHere

File: docs/transcription-project-guide/beta-testing/beta-step-by-step.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## 5.2.1 Beta checklist
+
+When you are ready for beta review, go through the checklist found in the Visibility tab of the Project Builder. At this stage, your project should be ready for volunteers to use – don’t submit something for beta testing that is unfinished, or that you wouldn’t send to the public. 
+
+
+## 5.2.2 Internal review
+
+When you initially apply for review, your project will first be reviewed by a member of the Zooniverse team. The team member will deliver their feedback using our standard [internal review form](https://docs.google.com/document/d/12l15ZDGrhWuWc1Nwy-v7sJeJb3JSjZNCqO_oCo_MluM/edit?usp=sharing). Once they have finished their review, the team member will write to you via a Talk post on a private (Team + Zooniverse) board they set up as part of the internal review process. **Make sure you are signed up to receive Talk emails for your project, or you may miss this step.** Once you have read the feedback, you can respond to the reviewer via the review form. Note that there is also a section for you to fill out, too.
```
Comment (eatyourgreens) [inline docs/transcription-project-guide/beta-testing/beta-step-by-step.md:16]: This is an example of accessible link text. 👍🏻

File: docs/transcription-project-guide/acknowledgements-and-resources/institute-website.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# 9.3 Institute website
+
+**CONTENT PLACEHOLDER**
```
Comment (eatyourgreens) [inline docs/transcription-project-guide/acknowledgements-and-resources/institute-website.md:3]: Is it ok to publish this with the placeholder text?

**Review by eatyourgreens — APPROVED** [review]
Looks good. Is it ok for this to go live with placeholder text?

**Comment by snblickhan** [comment]
@eatyourgreens Thanks for the link accessibility feedback -- I'll make a note to do a review of the link text throughout and update with this in mind.

FYI not planning to deploy immediately, just need to be able to preview so I can show my external collaborators (who are providing the text that currently has placeholder). Assigned to Cliff bc he's working on another component of this, and we discussed holding off on deploy in a call. Just wanted to include that here for clarity/so it doesn't get deployed before it's ready.

**Comment by eatyourgreens** [comment]
No problem. We don't actually have staging/preview for this, because it uses GitHub Pages.

**Comment by eatyourgreens** [comment]
By the way, if you change this PR to draft status, it can't be merged by mistake.

**Comment by snblickhan** [comment]
Sounds good. Cliff and I will chat about how to proceed.

File: docs/best-practices/introduction.md
Comment (eatyourgreens) [inline docs/best-practices/introduction.md:1]: Removing the directory index here will break links to https://help.zooniverse.org/best-practices/. Did you mean to do that?

File: docs/getting-started/how-to.md
Comment (eatyourgreens) [inline docs/getting-started/how-to.md:1]: Same comment here about removing the directory index. Each directory should have an index page as the entry point/landing page for that directory.

**Comment by lcjohnso** [comment]
Closing in favor of #75

**Pull request closed without merging**